### PR TITLE
Alerted processes

### PR DESCRIPTION
**Pull request by mitodrummer**
- ProcessAlerts details component added with rough UX to display rule and alert details.
- Processes in the tree now marked in red with expandable alerts button
- Fix to data routes to ensure proper ordering
- Util copied from another plugin to unflatten json (alerts coming out of .siem-signals-default index have some fields flattened...)
- Rule / alert fields that seemed useful were added to the ProcessEvent interface.

### CODE REVIEW

**Review by opauloh — APPROVED** [review]
lgtm

File: x-pack/plugins/session_view/public/components/ProcessTree/index.tsx [hunk 1]
```diff
@@ -105,7 +107,7 @@ export const ProcessTree = ({
         const isVisible = eTop >= cTop && eBottom <= cBottom;
 
         if (!isVisible) {
-          processEl.scrollIntoView();
+          // processEl.scrollIntoView();
```
Comment (zizhouW) [inline x-pack/plugins/session_view/public/components/ProcessTree/index.tsx:110]: should we keep this?

**Review by zizhouW — APPROVED** [review]
LGTM, just one comment

File: x-pack/plugins/session_view/public/components/ProcessTree/index.tsx (hunk 1 above)
Comment (mitodrummer) [reply x-pack/plugins/session_view/public/components/ProcessTree/index.tsx:110]: yup, just disabled for demo, needs tweaking.

**Pull request merged**